source/plugins/ruby/ApplicationInsightsUtility.rb

```diff
@@ -47,59 +49,43 @@ def initializeUtility()
       begin
         resourceInfo = ENV["AKS_RESOURCE_ID"]
         if resourceInfo.nil? || resourceInfo.empty?
-          @@CustomProperties["ACSResourceName"] = ENV[@@EnvAcsResourceName]
-          @@CustomProperties["ClusterType"] = @@AcsClusterType
-          @@CustomProperties["SubscriptionID"] = ""
-          @@CustomProperties["ResourceGroupName"] = ""
-          @@CustomProperties["ClusterName"] = ""
+          @@CustomProperties["ID"] = ENV[@@EnvAcsResourceName]
           @@CustomProperties["Region"] = ""
         else
-          @@CustomProperties["AKS_RESOURCE_ID"] = resourceInfo
-          begin
-            splitStrings = resourceInfo.split("/")
-            subscriptionId = splitStrings[2]
-            resourceGroupName = splitStrings[4]
-            clusterName = splitStrings[8]
-          rescue => errorStr
-            $log.warn("Exception in AppInsightsUtility: parsing AKS resourceId: #{resourceInfo}, error: #{errorStr}")
-          end
-          @@CustomProperties["ClusterType"] = @@AksClusterType
-          @@CustomProperties["SubscriptionID"] = subscriptionId
-          @@CustomProperties["ResourceGroupName"] = resourceGroupName
-          @@CustomProperties["ClusterName"] = clusterName
+          @@CustomProperties["ID"] = resourceInfo
           @@CustomProperties["Region"] = ENV[@@EnvAksRegion]
         end
 
         #Commenting it for now from initilize method, we need to pivot all telemetry off of kubenode docker version
         #getDockerInfo()
         @@CustomProperties["WorkspaceID"] = getWorkspaceId
-        @@CustomProperties["AgentVersion"] = ENV[@@EnvAgentVersion]
-        @@CustomProperties["ControllerType"] = ENV[@@EnvControllerType]
+        @@CustomProperties["Version"] = ENV[@@EnvAgentVersion]
+        @@CustomProperties["ControllerType"] = @@controllerType[ENV[@@EnvControllerType].downcase]
```

Comment:
Please double check these naming changes are updated every where.

Reply:
There are mentions in the telegraf.conf files in comments...Do we want to make changes there as well?

Reply:
> There are mentions in the telegraf.conf files in comments...Do we want to make changes there as well?

> we only need to change specific to telemetry. which fields are you referring in telegraf.conf? I see the ControllerType but its not commented out and also not related to telemetry.

Reply:
Talking about these: https://github.com/microsoft/Docker-Provider/blob/750b6b3a941c4b16e30c89c84c4372fa5581f5dd/build/linux/installer/conf/telegraf.conf#L19C1-L27C39

Reply:
> Talking about these: https://github.com/microsoft/Docker-Provider/blob/750b6b3a941c4b16e30c89c84c4372fa5581f5dd/build/linux/installer/conf/telegraf.conf#L19C1-L27C39

all these fields are commented out so we dont need to touch this file.